test/test_integration.py

```diff
@@ -6,39 +6,219 @@
 
 import pytest
 from retriever.lib.compile import compile_json
-from retriever.lib.parse_script_to_json import parse_script_to_json
 from retriever import HOME_DIR, ENGINE_LIST
 from retriever.lib.tools import file_2string
 from retriever.lib.tools import create_file
 
 simple_csv = {'name': 'simple_csv',
               'raw_data': "a,b,c\n1,2,3\n4,5,6\n",
-              'script': "shortname: simple_csv\ntable: simple_csv, http://example.com/simple_csv.txt",
+              'script': """{\n
+                        "name": "simple_csv",\n
+                        "resources": [\n
+                            {\n
+                                "dialect": {},\n
+                                "name": "simple_csv",\n
+                                "schema": {},\n
+                                "url": "http://example.com/simple_csv.txt"\n
+                            }\n
+                        ],\n
+                        "retriever": "True",\n
+                        "retriever_minimum_version": "2.0.0-dev",\n
+                        "script_version": 1.0,\n
```

Comment:
All of these `script_version`'s should be changed to `version` to match the recent update in the scripts.